go.mod

```
@@ -76,7 +76,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/util/pointer v0.49.0-rc.2
 	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.49.0-rc.2
 	github.com/DataDog/datadog-go/v5 v5.3.0
-	github.com/DataDog/datadog-operator v1.2.1
+	github.com/DataDog/datadog-operator v1.1.0
```

can we add a comment about why we should not update to 1.2.1

something like 
```suggestion
	// do not update datadog-operator to 1.2.1 because the indirect dependency github.com/DataDog/datadog-api-client-go/v2 v2.15.0 is trigger a huge Go heap memory increase.
	github.com/DataDog/datadog-operator v1.1.0
```